src/fiatconnect/saga.test.ts

```diff
@@ -1324,6 +1328,14 @@ describe('Fiatconnect saga', () => {
 
   describe('handleAttemptReturnUserFlow', () => {
     const selectProviderParams = {
+      amount: {
+        crypto: 2,
+        fiat: 2,
+      },
+      flow: CICOFlow.CashOut,
+      tokenId: mockCusdTokenId,
+    }
+    const selectProviderParamsReturn = {
```

Comment:
not sure I get this variable name, seems like its only used to construct attemptReturnUserFlow action? and is Network required?